Appwrite/Appwrite_User_Authentication.ipynb

```diff
@@ -103,16 +103,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "id": "81730041-2523-462e-83c7-f3c08a3ca960",
    "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-10-12T14:00:10.075663Z",
+     "iopub.status.busy": "2023-10-12T14:00:10.075439Z",
+     "iopub.status.idle": "2023-10-12T14:00:11.893615Z",
+     "shell.execute_reply": "2023-10-12T14:00:11.892973Z",
+     "shell.execute_reply.started": "2023-10-12T14:00:10.075640Z"
+    },
     "papermill": {},
     "tags": []
    },
    "outputs": [],
    "source": [
-    "from appwrite.client import Client\n",
-    "from appwrite.services.users import Users"
+    "try:\n",
+    "    from appwrite.client import Client\n",
+    "    from appwrite.services.users import Users\n",
+    "except ImportError:\n",
+    "    !pip install appwrite\n",
+    "    from appwrite.client import Client\n",
+    "    from appwrite.services.users import Users\n",
+    "try:\n",
+    "    import naas\n",
+    "except:\n",
```

Comment:
@MSaiKiran9 No need to install Naas, since it is pre-installed in the environment.